torch/testing/_internal/common_utils.py

```diff
@@ -614,8 +614,8 @@ def run_tests(argv=UNITTEST_ARGS):
         test_cases = discover_test_cases_recursively(suite)
         for case in test_cases:
             test_case_full_name = case.id().split('.', 1)[1]
-            other_args = (['--import-disabled-tests'] if IMPORT_DISABLED_TESTS else List[str]([]) +
-                          ['--import-slow-tests'] if IMPORT_SLOW_TESTS else List[str]([]))
+            other_args = list(('--import-disabled-tests',) if IMPORT_DISABLED_TESTS else () +
```

Comment:
Thanks for fixing this! Two questions.

Can this be written more simply as

```
other_args = []
if IMPORT_DISABLED_TESTS:
  other_args = ['--import-disabled-tests']
elif IMPORT_SLOW_TESTS:
  other_args = ['--import-slow-tests']
```
?

Follow-up question, this seems wrong, right? I would naively expect this to be written like:

```
other_args = []
if IMPORT_DISABLED_TESTS:
  other_args.append('--import-disabled-tests')
if IMPORT_SLOW_TESTS:
  other_args.append('--import-slow-tests')
```

Reply:
I have changed it to the latter one. I am not sure if mypy will allow me to do so. It is very sad that mypy often makes trivial things difficult.

Reply:
I'll exclude the file from mypy type checking

Reply:
Looks like mypy lets me go with that code :). But still, I do agree it is good to skip mypy on these files. My opinion is, mypy should cover all files that will be used by users but no files that are only used by developers.

Reply:
I don't think we can skip the mypy as it allows one to detect trivial regression like typos ahead of time, but we probably can disable mypy strict